### PR TITLE
Add com.alovoa.alovoa-electron

### PR DESCRIPTION
**Pull request by Nonononoki**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: com.alovoa.alovoa-electron.metainfo.xml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  <categories>
+    <category>Utility</category>
+  </categories>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://raw.githubusercontent.com//Alovoa/alovoa-electron/main/data/screenshots/screenshot-1.png</image>
+    </screenshot>
+  </screenshots>
+  <url type="bugtracker">https://github.com/Alovoa/alovoa-electron/issues</url>
+  <url type="homepage">alovoa.com</url>
```
Comment (hfiguiere) [inline com.alovoa.alovoa-electron.metainfo.xml:23]:
```suggestion
  <url type="homepage">https://alovoa.com</url>
```

It should be a valid URL.

File: com.alovoa.alovoa-electron.yml [hunk 2]
```diff
@@ -0,0 +1,81 @@
+app-id: com.alovoa.alovoa-electron
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
```
Comment (hfiguiere) [inline com.alovoa.alovoa-electron.yml:3]: 22.08 is the latest runtime.

File: com.alovoa.alovoa-electron.yml [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+  # Tray
+  - --talk-name=com.canonical.indicator.application
+  - --talk-name=org.ayatana.indicator.application
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --own-name=org.kde.*
+  # File system access
+  - --filesystem=home
+  - --filesystem=xdg-data/themes
+  - --filesystem=xdg-config/gtk-3.0
+  - --filesystem=xdg-config/gtk-4.0
```
Comment (hfiguiere) [inline com.alovoa.alovoa-electron.yml:31]: any reason why you need these 3 permission?

File: com.alovoa.alovoa-electron.yml [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+  - --filesystem=xdg-config/kdeglobals:ro
```
Comment (hfiguiere) [inline com.alovoa.alovoa-electron.yml:32]: or even this permission...

**Comment by hfiguiere** [comment]
bot, build com.alovoa.alovoa-electron

File: com.alovoa.alovoa-electron.yml (hunk 4 above)
Comment (Nonononoki) [reply com.alovoa.alovoa-electron.yml:32]: That's for detecting light/dark system themes on GNOME and KDE

**Comment by Nonononoki** [comment]
bot, build com.alovoa.alovoa-electron

**Comment by Nonononoki** [comment]
bot, build com.alovoa.alovoa-electron

File: com.alovoa.alovoa-electron.yml (hunk 3 above)
Comment (Nonononoki) [reply com.alovoa.alovoa-electron.yml:31]: That's for detecting light/dark system themes on GNOME and KDE

Comment (barthalion) [reply com.alovoa.alovoa-electron.yml:31]: I'm not sure if it works that way. There's `org.gnome.desktop.interface color-scheme` dconf key on GNOME, but Firefox detects dark system theme just fine without explicit access to GTK themes or host dconf. I'd prefer you to remove it.

File: com.alovoa.alovoa-electron.yml (hunk 2 above)
Comment (barthalion) [reply com.alovoa.alovoa-electron.yml:3]: 21.08 will be unmaintained in 6 months, so yeah, this needs updating to 22.08.

File: com.alovoa.alovoa-electron.yml (hunk 3 above)
Comment (Nonononoki) [reply com.alovoa.alovoa-electron.yml]: @barthalion Removed the above lines and updated the runtime version to 22.08

**Comment by Nonononoki** [comment]
bot, build com.alovoa.alovoa-electron

File: com.alovoa.alovoa-electron.yml [hunk 5]
```diff
@@ -0,0 +1,78 @@
+app-id: com.alovoa.alovoa-electron
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
```
Comment (barthalion) [inline com.alovoa.alovoa-electron.yml:6]: This one needs a bump too.

Comment (Nonononoki) [reply com.alovoa.alovoa-electron.yml]: Thank, updated the base version to 22.08

**Comment by Nonononoki** [comment]
bot, build com.alovoa.alovoa-electron

**Pull request closed without merging**